carbon-design-system/carbon-website
# feat: add data viz section to website

# Add Data Visualization section to site

closes: #240, #241, #364, #242, #243, #387, #351

## Dev review instructions 

A ton of files added because we are adding a whole new section to the left nav. A lot of them are images so do not need review. Please verify all the components used in the data viz pages are working:
- Tag component on Overview cards (new)
- InlineNotification component (new)
- Live chart demos on `Basic charts` and `Advanced charts` page
- Verify that all links and anchor links are working correctly
- Verify that all pages can be searched in search bar
- Verify that image / ArtDirection component display and response to breakpoints

We don't expect dev review to go into the content, but obviously if you see a problem with content, definitely tell us. Any additional feedback welcomed!

## Design review instructions

TBD - working with Mike in person on this at the moment. 

## Content review instructions

FYI @connor-leech @janchild 

- Make sure there are no conflicting guidance
- Clear messaging (eg. inline notification message)
- Heading, content, images and captions match
- Language, word choice, spelling, grammar help, including texts inside images
